### DISCUSSION TITLE
Generate date in iso8601 Format

### DISCUSSION THREAD

**Discussion by SethiShivam** [moment/moment#6428] [Q&A] [answered]
I want to generate date in iso8601 format 

which is giving result in format 

   **_YYYY-MM-DDTHH:MM:SS.ffffff+HH:MM[:SS[.ffffff]]
    .strftime('%Y-%m-%dT%H:%M:%S.%f%z')
    _**

exact format : **'2020-08-22T17:50:09.988921+00:00'**

I have tried but not able to generate the exact format .

Team please help me with this 

Best Regards
Sethi Shivam

**Comment by jeremywall** [comment]
the following format string produces something like what you have with the fractional seconds extended to 6 digits
YYYY-MM-DDTHH:mm:ss.SSSSSSZ

I'm in California to my current UTC offset is -7 hours and this format generates this output
2021-10-01T11:09:40.484000-07:00

**Comment by mattjohnsonpint** [comment, marked as answer]
Parsing is lenient unless strict mode is enabled. `moment('31-10-2021', 'YYYY-MM-DD', true).isValid()` returns `false`, while the matching `DD-MM-YYYY` format returns `true`.
